CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ add_dependencies(${PROJECT_NAME}
     )
 
 install(DIRECTORY launch
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/launch
+  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
```

Comment:
Yes, but in the first case there should be a slash at the end of directory name, i.e. `DIRECTORY launch/`. Otherwise it copies the directory itself, not just its contents, and that results in `share/<package>/launch/launch/` layout. Also, I think it is more clear this way, then using a slash in the path.